backend/app.py

```
@@ -160,12 +177,20 @@ async def search_items(search: SearchQuery):
-        return {"success": True, "items": items}
+        userItems = db_get_user_items(username)
+        print(f"DEBUG: get_user_items returning {len(userItems)} items for {username}")
+        return {"success": True, "items": userItems}
     except Exception as e:
         print(f"DEBUG: Exception in get_user_items: {e}")
         return {"success": False, "error": str(e), "items": []}
 
+def _noop_parse_int(s: str) -> int:
+      try:
+          return int(s)
+      except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)